html/elements/input/tel.json

```
@@ -19,7 +19,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": true
+                "version_added": "≤61"
```

I think this was in Firefox 4.

```suggestion
                "version_added": "4"
```